Functions/CreateCommand.py

```diff
@@ -0,0 +1,20 @@
+from command_factory import *
+from ICommand import ICommand
+import json
+import os
+import keys
+
+class CreateCommand(ICommand):
+    def __init__(self, schema):
+        self.schema = schema
+
+    def execute(self):
+        if self.schema is None:
+            return "Schema not found"
+        data = json.load(open(self.schema, 'r'))
+        path = os.path.join(os.getcwd(), data[keys.DATABASE])
+        os.makedirs(path, exist_ok=True)
+        for table in data[keys.TABLES]:
```

Comment:
if the schema doesn't have table keys, it will throw an exception here (NullPointerException)